frontend/src/pages/Profiles/Client/ClientArea.jsx

```diff
@@ -136,58 +181,41 @@ class ClientProfileArea extends React.Component {
                                 </Table.TextHeaderCell>
                                 <Table.TextHeaderCell>
-                                    Last Activity
+                                    Bidding Status
                                 </Table.TextHeaderCell>
-                                <Table.TextHeaderCell>ltv</Table.TextHeaderCell>
                             </Table.Head>
                             <Table.Body height={240}>
-                                <Table.Row intent="warning">
-                                    <Table.TextCell>
-                                        Jack Philips
-                                    </Table.TextCell>
-                                    <Table.TextCell>
-                                        Painting of my new village
-                                    </Table.TextCell>
-                                    <Table.TextCell>6 month ago</Table.TextCell>
-                                    <Table.TextCell isNumber>
-                                        $150
-                                    </Table.TextCell>
-                                </Table.Row>
-
-                                <Table.Row intent="danger">
-                                    <Table.TextCell>
-                                        Julia Williamson
-                                    </Table.TextCell>
-                                    <Table.TextCell>
-                                        Repairing my beautiful baby.
-                                    </Table.TextCell>
-                                    <Table.TextCell>
-                                        10 months ago
-                                    </Table.TextCell>
-                                    <Table.TextCell isNumber>
-                                        $250
-                                    </Table.TextCell>
-                                </Table.Row>
-
-                                <Table.Row intent="success">
-                                    <Table.TextCell>
-                                        Jonathan Martin
-                                    </Table.TextCell>
-                                    <Table.TextCell>
-                                        Giving my cat hugs
-                                    </Table.TextCell>
-                                    <Table.TextCell>2 moths ago</Table.TextCell>
-                                    <Table.TextCell isNumber>
-                                        $242
-                                    </Table.TextCell>
-                                </Table.Row>
+                                {this.state.jobs.map(job => (
+                                    <Table.Row
+                                        key={job.id}
+                                        isSelectable
+                                        onSelect={() => {
+                                            window.location = "/job/" + job.id;
```

Comment:
Instead of handling this with reload; we can handle it with client side redirect.

For that, you can add an `selectedJob` to state and when it is selected; render 
`<Redirect to={`/job/${selectedJob.id}`}/>`